recipes/lazypredict/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "lazypredict" %}
+{% set version = "0.2.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lazypredict-{{ version }}.tar.gz
+  sha256: 453853175517bd4b928c7122c39714688208f0e66dc4e9d7c1a945433f157158
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - lazypredict=lazypredict.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.6
+  run:
+    - click ==7.1.2
+    - joblib ==1.0.0
+    - lightgbm ==2.3.1
+    - numpy ==1.19.1
+    - pandas ==1.0.5
+    - pytest ==5.4.3
+    - python >=3.6
+    - pyyaml ==5.3.1
+    - scikit-learn ==0.23.1
+    - scipy >=1.5.3
+    - six ==1.15.0
+    - tqdm ==4.56.0
+    - xgboost ==1.1.1
```

Comment:
If the `xgboost` dependency were loosened this would probably pass on Windows...